.circleci/config.yml.j2

```diff
@@ -464,23 +500,41 @@ workflows:
       # {{ airflow_version }}
       {%- for distribution in distributions %}
       # {{ airflow_version }} - {{ distribution }}
+      - pull-image:
```

Comment:
I just tried scanning an image I don't have pulled and it worked. Do we even need this step?